core/controllers/suggestion.py

```diff
@@ -388,21 +400,11 @@ def post(self, suggestion_id):
             self.payload.get('question_state_data'))
         question_state_data_obj.validate(None, False)
 
-        updated_suggestion = suggestion_services.update_question_suggestion(
+        suggestion_services.update_question_suggestion(
```

Comment:
Can you explain conceptually what's happening here (why did the new_image_filenames code move)?

Reply:
OK -- could you please just check with whoever wrote the previous code why they had that code there originally? Just want to make absolutely certain that we aren't introducing a regression.